.github/workflows/test.yaml

```
@@ -149,11 +237,11 @@ jobs:
         python -m coverage xml -o coverage.xml -i
         python -m coverage report --skip-covered
 
-    - uses: codecov/codecov-action@v2
-      if: ${{ !cancelled() && contains(matrix['tox-env'], 'withcov') }}
+    - uses: codecov/codecov-action@v3
+      if: ${{ !cancelled() && !matrix['skip-coverage'] }}
       with:
         files: coverage.xml
-        name: lnx-${{ matrix.python-version }}-${{ matrix.tox-env }}${{ matrix.noipv6 }}
+        name: ${{ matrix.python-version || env.DEFAULT_PYTHON_VERSION }}-${{matrix.job-name || 'lnx-default-tests' }}
```

I'm not sure `name` even does anything here (I can't see this name anywhere in the UI except where the config itself is being dumped) but ...

```suggestion
        name: ${{ matrix.python-version || env.DEFAULT_PYTHON_VERSION }}-${{matrix.job-name || 'default-tests' }}
```

enjoy github's delightful decision to render this suggestion as latex or some crazy thing

---

This is for the Codecov reports UI.
I was using it to debug Codecov, as a way to check if the coverage report from a build was received by Codecov.